spec/intl.html

```diff
@@ -0,0 +1,372 @@
+          1. If _hc_ is *null*, then
+            1. Set _hc_ to _hcDefault_.
+          1. If _hour12_ is not *undefined*, then
+            1. If _hour12_ is *true*, then
+              1. If _hcDefault_ is `"h11"` or `"h23"`, then
+                1. Set _hc_ to `"h11"`.
+              1. Else,
+                1. Set _hc_ to `"h12"`.
+            1. Else,
+              1. Assert: _hour12_ is *false*.
+              1. If _hcDefault_ is `"h11"` or `"h23"`, then
+                1. Set _hc_ to `"h23"`.
+              1. Else,
+                1. Set _hc_ to `"h24"`.
+          1. Set _dateTimeFormat_.[[HourCycle]] to _hc_.
+          1. If _dateTimeformat_.[[HourCycle]] is `"h11"` or `"h12"`, then
+            1. Let _pattern_ be _bestFormat_.[[pattern12]].
+          1. Else,
+            1. Let _pattern_ be _bestFormat_.[[pattern]].
+        1. Else,
+          1. Set _dateTimeFormat_.[[HourCycle]] to *undefined*.
+          1. Let _pattern_ be _bestFormat_.[[pattern]].
+        1. Set _dateTimeFormat_.[[Pattern]] to _pattern_.
+        1. <ins>For each row in <emu-xref href="#table-temporal-patterns"></emu-xref>, except the header row, in table order, do</ins>
+          1. <ins>Let _limitedOptions_ be a new Record.</ins>
+          1. <ins>Let _fields_ be the list of fields in the Fields column of the row.</ins>
+          1. <ins>For each field _field_ of _opts_, do</ins>
+            1. <ins>If _field_ is in _fields_, then</ins>
+              1. <ins>Set _limitedOptions_.[[&lt;field>]] to _opts_.[[&lt;field>]].</ins>
+          1. <ins>If _matcher_ is `"basic"`, then</ins>
+            1. <ins>Let _bestFormat_ be BasicFormatMatcher(_limitedOptions_, _formats_).</ins>
+          1. <ins>Else,</ins>
+            1. <ins>Let _bestFormat_ be BestFitFormatMatcher(_limitedOptions_, _formats_).</ins>
+          1. <ins>If _bestFormat_ has any fields that are in _fields_, then</ins>
+            1. <ins>If _bestFormat_ has an [[hour]] field, then</ins>
+              1. <ins>Do the hourCycle dance.</ins>
+            1. <ins>Else,</ins>
+              1. <ins>Let _pattern_ be _bestFormat_.[[pattern]].</ins>
+          1. <ins>Else,</ins>
+            1. <ins>Let _pattern_ be *null*.</ins>
+          1. <ins>Set _dateTimeFormat_'s internal slot whose name is the Pattern column of the row to _pattern_.</ins>
+        1. Return _dateTimeFormat_.
+      </emu-alg>
+
+      <emu-table id="table-temporal-patterns">
+        <emu-caption><ins>Supported fields for temporal patterns</ins></emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Pattern</th>
+              <th>Fields</th>
+            </tr>
+          </thead>
+          <tr>
+            <th>[[CivilDatePattern]]</td>
+            <td>[[weekday]], [[era]], [[year]], [[month]], [[day]]</td>
+          </tr>
+        </table>
+      </emu-table>
+
+    </emu-clause>
+
+    <emu-clause id="sec-formatnumberstring" aoid="IsTemporalObject">
```

Comment:
If you add this here, it changes the section numbers for all following sections.  Probably safest to just add new operations at the end.